labonneboite/web/static/css/vendor/bootstrap-tooltip.css

```diff
@@ -5,119 +5,119 @@
 */
 /*!
 * Generated using the Bootstrap Customizer (http://getbootstrap.com/customize/?id=10b3f9c39013afc13145d3e4eb1ff982)
-* Config saved to config.json and https://gist.github.com/10b3f9c39013afc13145d3e4eb1ff982
+* Config saved to bootstrap-tooltip-config.json
 */
 .tooltip {
-    position: absolute;
-    z-index: 1070;
-    display: block;
-    font-family: "Helvetica Neue", Helvetica, Arial, sans-serif;
-    font-style: normal;
-    font-weight: normal;
-    letter-spacing: normal;
-    line-break: auto;
-    line-height: 1.42857143;
-    text-align: left;
-    text-align: start;
-    text-decoration: none;
-    text-shadow: none;
-    text-transform: none;
-    white-space: normal;
-    word-break: normal;
-    word-spacing: normal;
-    word-wrap: normal;
-    font-size: 12px;
-    opacity: 0;
-    filter: alpha(opacity=0);
+  position: absolute;
+  z-index: 1070;
+  display: block;
+  font-family: "Helvetica Neue", Helvetica, Arial, sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  letter-spacing: normal;
+  line-break: auto;
+  line-height: 1.42857143;
+  text-align: left;
+  text-align: start;
+  text-decoration: none;
+  text-shadow: none;
+  text-transform: none;
+  white-space: normal;
+  word-break: normal;
+  word-spacing: normal;
+  word-wrap: normal;
+  font-size: 12px;
+  opacity: 0;
+  filter: alpha(opacity=0);
 }
 .tooltip.in {
-    opacity: 0.9;
-    filter: alpha(opacity=90);
+  opacity: 1;
+  filter: alpha(opacity=100);
```

Comment:
`Note: IE8 and earlier use filter:alpha(opacity=x). The x can take a value from 0 - 100. A lower value makes the element more transparent.` (https://www.w3schools.com/css/css_image_transparency.asp)
Je pense qu'on peut s'en passer :laughing:

Reply:
Ce fichier est auto-généré par bootstrap, moi je n'y ai pas touché (et je pense qu'il vaut mieux éviter)

Reply:
Je n'avais pas compris que c'éatit un code auto-généré.
Je reste sceptique concernant les codes auto-générés (et en plus, on doit conserver le fichier `bootstrap-toolkit-config.json`...). Mais bon, je comprends que ça permette de gagner du temps... donc on va dire ok !

Reply:
Je suis bien d'accord avec toi : s'il est auto-généré, c'est qu'il n'a pas besoin d'être versionné. Mais on n'utilise qu'un morceau de bootstrap et on ne peut pas récupérer ce morceau de bootstrap au moment du build. Auparavant, la config bootstrap était stockée sur github sous la forme d'un gist ; ce n'est plus le cas, il faut donc bien la stocker quelque part. En écrivant ça, je me rends compte qu'on pourrait stocker la config dans un gist appartenant à StartupsPoleEmploi plutôt que de la stocker dans le dépôt. Mais on court le risque que quelqu'un le supprime par inadvertance. C'est comme tu veux.

Reply:
On va le garder ici, ce sera plus simple :) C’est juste le concept qu'on utilise du code généré qui me perturbe....